collector/collector.py

```diff
@@ -356,7 +356,7 @@ def get_health():
 
 # Starts the web server and listen on all external IPs associated with this
```

Comment:
Please add in the file comment:

Runs the cluster insight data collector in master mode.